metapool/sample_sheet.py

```diff
@@ -1366,13 +1364,32 @@ class TellseqMetagSampleSheetv10(KLTellSeqSampleSheet):
     _HEADER[_SHEET_VERSION_KEY] = '10'
     _HEADER[_ASSAY_KEY] = _METAGENOMIC
 
+    @property
+    def CARRIED_PREP_COLUMNS(self):
+        return [x for x in _BASE_CARRIED_PREP_COLUMNS if x not in
+                {'i7_index_id', 'index', 'index2', 'i5_index_id'}]
+
+    @property
+    def GENERATED_PREP_COLUMNS(self):
+        return list(_BASE_GENERATED_PREP_COLUMNS)
+
```

Comment:
I am almost 100% certain that it is *not* necessary to add the GENERATED_PREP_COLUMNS property because it already exist for these classes!  The base `KLSampleSheet` class exposes a `GENERATED_PREP_COLUMNS` property, and that property is inherited down from `KLSampleSheet` to `KLSampleSheetWithSampleContext` to `KLTellSeqSampleSheet` to the specific tellseq version sheets.  Using the existing master (without this PR), I can see this property  on a `TellseqMetagSampleSheetv10` instance:

<img width="947" alt="Screenshot 2024-12-10 at 2 23 40 PM" src="https://github.com/user-attachments/assets/74b70a96-55c3-4475-b921-432b4a84deca">

Reply:
ok, works for me.